infra/media/KThumbnailCapture.php

```diff
@@ -18,9 +18,10 @@ public static function shouldResizeByPackager($params, $type, $dimension)
 	}
 
 
-	public static function generateThumbUrlWithOffset($url, $calc_vid_sec, $packagerCaptureUrl, $capturedThumbPath, $width = null, $height = null, $offsetPrefix = '', $postFix = '')
+	public static function generateThumbUrlWithOffset($url, $calc_vid_sec, $packagerCaptureUrl, $capturedThumbPath, $width = null, $height = null, $offsetPrefix = '', $postFix = '', $offsetPostfix='')
 	{
 		$offset = floor($calc_vid_sec * 1000);
+		$offset .= $offsetPostfix;

```

Comment:
let concat to `$offset = $offsetPrefix . $offset . $offsetPostfix;`
And separate to small function for build the string